src/control/cmd/daos_server/auto_test.go

```diff
@@ -260,6 +258,30 @@ func TestDaosServer_Auto_confGen(t *testing.T) {
 			WithTargetCount(18).WithHelperStreamCount(4),
 	}
 
+	var defCoresPerNuma uint32 = 26
```

Comment:
can't we query the CPU's actual #cores instead of hardcoding a number? what if we run on a server that only has 16 cores/CPU socket?  or 128?  or what if we have sub-numa clustering enabled so we have more than one numa domain per physical cpu socket?

Reply:
From my understanding, this is only for unit testing purpose.